papers/2024-morganton-srinath/myst.yml

```diff
@@ -0,0 +1,35 @@
+version: 1
+project:
```

Comment:
Add DOI ignore statements like this:

https://github.com/scipy-conference/scipy_proceedings/blob/82a60c3fedc6fb3eec6df8c00fe0b7812bc4ae8a/papers/00_myst_template/myst.yml#L36C89-L45C19

This allows the pull-request to go green!

Reply:
Made that change, but it looks like it's still not ignoring the lack of a DOI.